### PR TITLE
Scenario6

### PR DESCRIPTION
**Pull request by irina-pereiaslavskaia**

### CODE REVIEW

File: csm_test_utils/rds/rds_backup.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import json
+
+from influx_line_protocol import Metric, MetricCollection
+from ocomone import setup_logger
+from requests import Response, HTTPError
+from datetime import datetime
+from ..common import Client, base_parser, sub_parsers
+
+
+BASE_URL = "https://rds.eu-de.otc.t-systems.com/v3"
```
Comment (anton-sidelnikov) [inline csm_test_utils/rds/rds_backup.py:16]: pass this as arg

File: csm_test_utils/rds/rds_backup.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+CSM_EXCEPTION = "csm_exception"
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+
+
+def get_auth_token(cloud_config, cloud_name) -> str:
+    """Get auth token using data from clouds.yaml file. Token and project_id are returned as a string"""
+    with open(cloud_config) as clouds_yaml:
+        data = yaml.safe_load(clouds_yaml)
+    request_headers = {'Content-Type': 'application/json;charset=utf8'}
```
Comment (anton-sidelnikov) [inline csm_test_utils/rds/rds_backup.py:27]: make variable for 'Content-Type': 'application/json;charset=utf8'

File: csm_test_utils/rds/rds_backup.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    url = "/".join([BASE_URL, "auth/tokens"])
+    response = requests.post(url = url, data = request_body, headers = request_headers)
+    token = response.headers.get('X-Subject-Token')
+    project_id = response.json()['token']['project']['id']
+    return token, project_id
+
+
+def get_rds_backup_info(token: str, project_id: str, **request_params) -> Response:
+    url = "/".join([BASE_URL, project_id, "backups?"])
+    request_headers = {'Content-Type': 'application/json;charset=utf8', 'X-Auth-Token': token}
```
Comment (anton-sidelnikov) [inline csm_test_utils/rds/rds_backup.py:58]: 'Content-Type': 'application/json;charset=utf8' - same

File: csm_test_utils/rds/rds_backup.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    args, _ = AGP.parse_known_args()
+    token, project_id = get_auth_token(args.cloud_config, args.cloud_name)
+    request_params = {'instance_id': args.instance_id, 'backup_type': 'auto'}
+    client = Client(args.target, args.telegraf)
+    setup_logger(LOGGER, "rds_backup_monitor", log_dir = args.log_dir, log_format = "[%(asctime)s] %(message)s")
+    LOGGER.info(f"Started monitoring of {client.url} (telegraf at {client.tgf_address})")
+    while True:
+        try:
+            report(client, token, project_id, **request_params)
+            time.sleep(60)
```
Comment (anton-sidelnikov) [inline csm_test_utils/rds/rds_backup.py:125]: once per minute is too often, once per hour on more is ok

**Review by Polina-Gubina — APPROVED** [review]

File: csm_test_utils/rds/rds_backup.py (hunk 1 above)
Comment (irina-pereiaslavskaia) [reply csm_test_utils/rds/rds_backup.py]: Done, pass endpoint as arg, add api version as const

File: csm_test_utils/rds/rds_backup.py (hunk 2 above)
Comment (irina-pereiaslavskaia) [reply csm_test_utils/rds/rds_backup.py]: Done

File: csm_test_utils/rds/rds_backup.py (hunk 3 above)
Comment (irina-pereiaslavskaia) [reply csm_test_utils/rds/rds_backup.py]: Add content type as const

File: csm_test_utils/rds/rds_backup.py (hunk 4 above)
Comment (irina-pereiaslavskaia) [reply csm_test_utils/rds/rds_backup.py]: Done

**Review by Polina-Gubina — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**